msr/msfr/core_cfd/3d_fullcore_les/SIZE

```diff
@@ -0,0 +1,42 @@
+c
+c     Include file to dimension static arrays
+c     and to set some hardwired run-time parameters
+c
+      integer ldim,lx1,lxd,lx2,lx1m,lelg,lelt,lpmin,ldimt
+      integer lpelt,lbelt,toteq,lcvelt
+      integer lelx,lely,lelz,mxprev,lgmres,lorder,lhis
+      integer maxobj,lpert,nsessmax,lxo
+      integer lfdm,ldimt_proj,lelr
+
+      ! BASIC
+      parameter (ldim=3)               ! domain dimension (2 or 3)
+      parameter (lx1=8)                ! GLL points per element along each direction
+      parameter (lxd=lx1/2*3)          ! GL  points for over-integration (dealiasing) 
+      parameter (lx2=lx1-0)            ! GLL points for pressure (lx1 or lx1-2)
+                                     
+      parameter (lelg=616000)          ! max number of global elements
+      parameter (lpmin=32*32)          ! min number of MPI ranks 
+      parameter (lelt=lelg/lpmin + 3)  ! max number of local elements per MPI rank
+      parameter (ldimt=1)              ! max auxiliary fields (temperature + scalars)
+
+      ! OPTIONAL
+      parameter (ldimt_proj=1)         ! max auxiliary fields residual projection
+      parameter (lelr=lelg)            ! max number of local elements per restart file
+      parameter (lhis=1)               ! max history/monitoring points
+      parameter (maxobj=1)             ! max number of objects
+      parameter (lpert=1)              ! max number of perturbations
+      parameter (toteq=1)              ! max number of conserved scalars in CMT
+      parameter (nsessmax=2)           ! max sessions to NEKNEK
```

Comment:
what is NEKNEK

Reply:
The parameter can be ignored here. FYI, Neknek is a native Nek coupling utility that can couple two Nek simulations on separate meshes/subdomains. The two subdomains can be meshed individually.